dts/bindings/ethernet/qca,ar8031.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Qualcomm AR8031 PHY
+
+compatible: "qca,ar8031"
+
+include: phy.yaml
+
+properties:
+  reg:
+    required: true
+    description: PHY address
+
+  EEE:
+    type: boolean
+    description: Energy Efficient Ethernet
```

Comment:
this description is not sufficient to describe what this property does

Reply:
Updated, thanks.